### DISCUSSION TITLE
357 rubblegloveCHROME.bmp texture don't have rt chrome

### DISCUSSION THREAD

**Discussion by GreenGroove46** [sultim-t/xash-rt#137] [Q&A]
RubblegloveCHROME.bmp texture have gray texture without textures, i tryed to rename rtx rubbleglove material but it not help.

**Comment by THG327** [comment]
It's meant to be replaced with a shader that uses ray tracing instead but if the texture name in the model is different then the shader doesn't get replaced. 
You can use a hex editor such as HxD to rename the texture within the model to match. In my case, I had to rename the texture in the model from "rubbergloveCHROME" to "RubbergloveCHROME" for a modded fixed weapon pack

[v_357.zip](https://github.com/sultim-t/xash-rt/files/10944322/v_357.zip)

**Reply by sultim-t** [reply]
Texture properties are specified in the `rt/data/textures.json`, so modifying that text file should be easier than modifying the original model.
More about it here https://github.com/sultim-t/xash-rt/discussions/124
